docs/source/models/concept_models/vivarium_mncnh_portfolio/anc_detail_module/module_document.rst

```diff
@@ -0,0 +1,136 @@
+.. role:: underline
+    :class: underline
+
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  +++++++++++++++++++++
+
+  Section Level 2 (#.#)
+  ---------------------
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+.. _2024_vivarium_mncnh_portfolio_anc_detail_module:
+
+======================================
+Antenatal care detail module
+======================================
+
+.. contents::
+  :local:
+  :depth: 2
+
+1.0 Overview
+++++++++++++
+
+This module builds upon the :ref:`antenatal care attendance module <2024_vivarium_mncnh_portfolio_anc_module>` that determines whether or not a simulant attends an ANC visit at any point during pregnancy and additionally determines the frequency and timing of those visits.
+
+2.0 Module Diagram and Data
++++++++++++++++++++++++++++++++
+
+2.1 Module Diagram
+----------------------
+
+.. image:: anc_detail_module_diagram.png
+
+2.2 Module Inputs
+---------------------
+
+.. list-table:: Module required inputs
+  :header-rows: 1
+
+  * - Input
+    - Source module
+    - Application
+    - Note
+  * - ANC attendance
+    - :ref:`ANC attendance module <2024_vivarium_mncnh_portfolio_anc_module>`
+    - Used to determine answer to decision node #1
+    - 
+  * - Pregnancy term duration
+    - :ref:`Pregnancy I <2024_vivarium_mncnh_portfolio_pregnancy_module>`
+    - Used to determine answer to decision node #2
+    - 
+
+
+2.3 Module Decision Nodes
+-----------------------------
+
+.. list-table:: Module decision nodes
+  :header-rows: 1
+
+  * - Decision node
+    - Description
+    - Information
+    - Note
+  * - 1
+    - Attends ANC?
+    - Informed from ANC attendance input value
+    - True/False
+  * - 2
+    - Partial term pregnancy?
+    - Informed from pregnancy duration input value
+    - "Yes" if input = 'partial', "No" if input = 'full'
+  * - 3
+    - Attends 4 ANC visits?
+    - Probability equal to ANC4 rates (GBD covariate ID 8): :code:`get_covariate_estimates(location_id=location_id, gbd_round_id=7, year_id=2021, decomp_step='iterative', covariate_id=8)`
+    - 
+
+.. todo::
+
+  Confirm if we want to be using GBD 2023 values here?? Note we have used GBD 2021 for existing implementation of ANC1 
```

Comment:
We will be updating the whole model to GBD 2023 in this quarter, so might be worth it to start that sooner.